packages/client/components/TaskFooterIntegrateMenu.tsx

```diff
@@ -131,6 +137,8 @@ const TaskFooterIntegrateMenu = (props: Props) => {
       mutationProps={mutationProps}
       teamId={teamId}
       label={label}
+      gitlabRef={viewerIntegrations.gitlab}
```

Comment:
I don't love that this is so GitLab specific, but we do need the GitLab cloud provider info to be able to complete the auth [here](https://github.com/ParabolInc/parabol/blob/e11747dfe132959d3ff09221debdb2398e2b4c66/packages/client/components/AddToGitLabMenuItem.tsx#L53). 

I considered passing down `integrationsRef` instead, but as we're just using GitLab here, I felt it was best to pass down the smallest amount of data possible.

Reply:
Maybe also consider passing `cloudProvider` which should be generic across different integrations

Reply:
I like it and agree that it's more generic, but `cloudProvider` can be null so `cloudProviderRef` won't work as the ref in the child fragment